stdlib/public/stubs/CommandLine.cpp

```diff
@@ -58,6 +58,11 @@ extern "C" char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
   return *_NSGetArgv();
 }
 #elif defined(__linux__) || defined(__CYGWIN__) || defined(__FreeBSD__)
+#if defined(__FreeBSD__)
```

Comment:
Since `/proc` is optional and not mounted by default, I'd recommend using `sysctl(3)` here. The MIB you want is `{ CTL_KERN, KERN_PROC, KERN_PROC_ARGS, getpid() }`